core/src/main/scala/org/apache/spark/sql/TiStrategy.scala

```diff
@@ -80,6 +80,10 @@ case class TiStrategy(getOrCreateTiContext: SparkSession => TiContext)(sparkSess
   private def allowIndexRead(): Boolean =
     sqlConf.getConfString(TiConfigConst.ALLOW_INDEX_READ, "true").toLowerCase.toBoolean
 
+  private def allowTiFlashRead(): Boolean = {
+    sqlConf.getConfString(TiConfigConst.USE_TIFLASH, "true").toLowerCase.toBoolean
```

Comment:
`Why` is it default as true?
What is its difference with `TiRowRDD.scala:43`?

Reply:
It should be false.